ci/README.md

```diff
@@ -107,6 +117,72 @@ The subsequent PRs on community will be able to pass the
 `check-merge-to-enterprise` check as long as they don't have a conflict with
 `enterprise-master`.
 
+### What to do when your branch got outdated?
+
+So there's one issue that can occur. Your branch will become outdated with
+master and you have to make it up to date. There's two ways to do this using
+`git merge` or `git rebase`. As usual, `git merge` is a bit easier than `git
+rebase`, but clutters git history. This section will explain both. If you don't
+know which one makes the most sense, start with `git rebase`, if for whatever
+reason this doesn't work feel free to fall back to `git merge`.
+
+#### Updating both branches with `git rebase`
+
+In the community repo:
+
+```bash
+git checkout $PR_BRANCH
+git pull origin master --rebase
+git push --force-with-lease
+```
+
+In the enterprise repo:
+
+```bash
+git checkout $PR_BRANCH
```

Comment:
Since both community and enterprise have the branch with the same name, it is best to set origin here.

Reply:
This is merely to ensure you are on the right branch, so that this code is copy-pastable. Changing it to `git checkout origin/$PR_BRANCH` would checkout the commit, but not the branch. So that would not work. Maybe you mean something else and I am misunderstanding.

Reply:
I think of `git checkout -b $PR_BRANCH -t origin/$PR_BRANCH` which fails if there is already a local branch with the given name.

I can not think of a better alternative, so we can keep it as it is